README.md

```diff
@@ -4,6 +4,26 @@ Currently supports Linux C/C++ (including CGO) & Java builds on Ubuntu 16.04 or
 
 ## Usage instructions
 
+For the most up-to-date instructions, see our [Official Documentation](https://buildbuddy.io/docs/rbe-setup).
+
+#### BzlMod
+
+Add the following to your `MODULE.bazel` file:
+
+```python
+bazel_dep(name = "toolchains_buildbuddy")
+
+buildbuddy = use_extension("@buildbuddy//:extensions.bzl", "buildbuddy")
+
+use_repo(buildbuddy, "buildbuddy_toolchain")
```

Comment:
Should we recommend this by default? It doesn't do anything by itself unless users would commonly want to reference the `buildbuddy_toolchain` repo.

Reply:
That's a good call. Now that we have the alias in place, we can just use the repo as a transitive dep instead of a direct dep.